pandas/tests/io/test_sql.py

```diff
@@ -479,6 +479,25 @@ def _transaction_test(self):
         res2 = self.pandasSQL.read_query('SELECT * FROM test_trans')
         assert len(res2) == 1
 
+    def _test_insert_multivalues(self):
```

Comment:
Add a comment with the Github issue number.

Reply:
And since this is sqlalchemy-specific, could you just define the test directly on `_TestSQLAlchemy`?

Reply:
Added github issues as comments and test moved